Tests/SwiftParserTest/DeclarationTests.swift

```diff
@@ -190,7 +198,11 @@ final class DeclarationTests: XCTestCase {
         DiagnosticSpec(locationMarker: "1️⃣", message: "unexpected code '{}' before enum case"),
         DiagnosticSpec(locationMarker: "2️⃣", message: "expected identifier in enum case", fixIts: ["insert identifier"]),
         DiagnosticSpec(locationMarker: "2️⃣", message: "expected '}' to end protocol", fixIts: ["insert '}'"]),
```

Comment:
I think you are running into the `FixItApplier` problem that I mentioned before here. We aren’t actually applying the Fix-It to insert the `}`.

Reply:
https://github.com/apple/swift-syntax/issues/1600